ADAL/tests/app/src/mac/ADTestAppAcquireTokenWindowController.h

```diff
@@ -29,8 +29,8 @@
 @interface ADTestAppAcquireTokenWindowController : NSWindowController
 {
     IBOutlet NSView* _authView;
-    IBOutlet WebView* _webView;
     
+    IBOutlet NSView *_contentWebView;
```

Comment:
nit: can we call it contentView or webviewContentView?

Reply:
It has 2 "views", and still sound ugly :p thus, I refuse to change this